pkg/db/db.go

```diff
@@ -1596,6 +1812,64 @@ func (h *DBHandler) RunCustomMigrationQueuedApplicationVersions(ctx context.Cont
 	})
 }
 
+// For commit_events migrations, we need some transformer to be on the database before we run their migrations.
+func (h *DBHandler) RunCustomMigrationsEventSourcingLight(ctx context.Context) error {
+	return h.WithTransaction(ctx, func(ctx context.Context, transaction *sql.Tx) error {
+		l := logger.FromContext(ctx).Sugar()
+		eslEvent, err := h.DBReadEslEventInternal(ctx, transaction, false)
+		if err != nil {
+			l.Warnf("could not get applications from database - assuming the manifest repo is correct: %v", err)
+			return err
+		}
+		if eslEvent != nil {
+			return nil
+		}
+		span, _ := tracer.StartSpanFromContext(ctx, "RunCustomMigrationsEventSourcingLight")
```

Comment:
this span sounds like it should go over the whole function `RunCustomMigrationsEventSourcingLight`.